libs/python.json

```diff
@@ -75,6 +78,7 @@
   {"id":"py.kivy","imports":["kivy"],"name":"kivy","repo":"kivy/kivy","tags":["framework"],"tech":["gui"],"status":"model-ready"},
   {"id":"py.pil","imports":["PIL"],"name":"PIL","repo":"python-pillow/Pillow","tags":["manipulation"],"tech":["images"],"status":"model-ready"},
   {"id":"py.aiohttp","imports":["aiohttp"],"name":"aiohttp","repo":"aio-libs/aiohttp","tags":["http-framework"],"tech":["internet"],"status":"awaiting-model"},
+  {"id":"py.cherrypy","imports":["CherryPy"],"name":"CherryPy","repo":"cherrypy/cherrypy","tags":["http-server"],"tech":["internet"],"status":"awaiting-model"},
```

Comment:
tags == http-framework maybe? that's how they describe themselves.

Reply:
oh, I thought they should be unified close as possible, will change it